src/libs/ajax/leonardo/Disks.ts

```diff
@@ -1,32 +1,34 @@
 import _ from 'lodash/fp'
 import * as qs from 'qs'
 import { appIdentifier, authOpts, fetchLeo, jsonBody } from 'src/libs/ajax/ajax-common'
-import { pdTypes } from 'src/pages/workspaces/workspace/analysis/utils/disk-utils'
+import { DecoratedPersistentDisk, GetDiskItem, ListDiskItem } from 'src/libs/ajax/leonardo/models/disk-models'
+import { updatePdType } from 'src/pages/workspaces/workspace/analysis/utils/disk-utils'
 
 
 export const Disks = signal => ({
-  list: async (labels = {}) => {
+  list: async (labels = {}): Promise<ListDiskItem> => {
     const res = await fetchLeo(`api/google/v1/disks${qs.stringify(labels, { addQueryPrefix: true })}`,
       _.mergeAll([authOpts(), appIdentifier, { signal }]))
     return res.json()
   },
 
-  disk: (project, name) => {
+  disk: (project: string, name: string) => {
     return {
```

Comment:
can this return be shorthanded to `disk: (project: string, name: string) => ({returnObject})` ?